### PR TITLE
Online feature store samples

### PR DESCRIPTION
**Pull request by ynpandey**
# Description
Added online feature store sample notebook and supporting files.

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/featurestore_sample/project/fraud_model/online_inference/src/scoring.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    global features
+
+    featurestore = FeatureStoreClient(credential=credential)
+
+    features = featurestore.resolve_feature_retrieval_spec(spec_path)
+
+    init_online_lookup(features, credential)
+
+    time.sleep(20)
```
Comment (Adarsh-Ramanathan) [inline sdk/python/featurestore_sample/project/fraud_model/online_inference/src/scoring.py:48]: nit: Why are we waiting here?

**Review by Adarsh-Ramanathan — APPROVED** [review]
LGTM overall, left one comment about a `time.sleep` in the scoring script.

File: sdk/python/featurestore_sample/project/fraud_model/online_inference/src/scoring.py (hunk 1 above)
Comment (ynpandey) [reply sdk/python/featurestore_sample/project/fraud_model/online_inference/src/scoring.py:48]: As per comments elsewhere in a similar example _It may take ~10 sec or so for the local RPC server of online store client to be ready_.

Comment (Adarsh-Ramanathan) [reply sdk/python/featurestore_sample/project/fraud_model/online_inference/src/scoring.py:48]: Yeah, but why the wait? `init_online_lookup` might take 10 seconds to return, but once it does, the server is initialized and ready to go.

**Pull request merged**